src/style.css

```diff
@@ -0,0 +1,68 @@
+.boxes {
+  color: rgb(255, 255, 255);
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  display: grid;
+  grid-template: "box-1" 300px
+    "box-2" 300px
+    "box-3" 300px
+    "box-4" 300px
+    "box-5" 300px
+    "box-6" 300px;
+}
+
+.boxes__box-1,
```

Comment:
add a general class name for blocks you want to have similar styles and style it

Reply:
Thank you, done.